Applications/Spire/Include/Spire/Charting/TrendLineModel.hpp

```diff
@@ -88,6 +96,7 @@ namespace Spire {
       std::vector<TrendLineEntry> m_trend_lines;
 
       std::vector<TrendLineEntry>::iterator find_id(int id);
+      bool is_selected(int id);
```

Comment:
Why isn't this method ```const```?

Reply:
It uses find_if(), which isn't const. This came up before for another function and you said to just make it non-const.

Reply:
```find_if``` is ```const```.